meson_options.txt

```diff
@@ -41,10 +41,10 @@ option(
 )
 
 option(
-    'experimental_mmx',
+    'emulate_mmx',
     type: 'boolean',
-    value: false,
-    description: 'Enable experimental Pentium MMX CPU emulation',
+    value: true,
+    description: 'Enable Pentium MMX CPU emulation',
```

Comment:
What's the thinking behind keeping the flag for this, given it's not experimental anymore and this is not an external dependency? Caution in case we need to roll back / disable it?